tests/conftest.py

```diff
@@ -195,6 +207,16 @@ def eos():
         return eos
 
 
+@pytest.fixture(scope='session')
+def ansible_inventory():
+    ansible_root = os.path.realpath(os.path.join(os.path.dirname(__file__), "../ansible"))
+    inventory_file_name = os.path.join(ansible_root, "inventory")
+
+    data_loader = DataLoader()
+    inventory = InventoryManager(loader = data_loader, sources=[inventory_file_name])
+    yield inventory
```

Comment:
This fixture is not necessary. Through the `duthost` or `ptfhost` object, we can get access to object instance of InventoryManager: 
```
duthost.host.options['inventory_manager']
```

Another issue is that this fixture assume default inventory file `ansible/inventory`. This is not the case for all vendors.

Reply:
Fixed